.github/ISSUE_TEMPLATE/BUG_REPORT.yml

```diff
@@ -0,0 +1,78 @@
+name: Bug Report
+description: Something isn't working as expected
+labels: [bug]
+body:
+- type: markdown
+  attributes:
+    value: |
+      Thank you for contributing to Swift-DocC! 
+      
+      Before you submit your issue, please complete each text area below with the relevant details for your bug, and complete the steps in the checklist
+- type: textarea
+  attributes:
+    label: Description
+    description: |
+      A short description of the incorrect behavior. 
+
+      If you think this issue has been recently introduced and did not occur in an earlier version, please note that. If possible, include the last version that the behavior was correct in addition to your current version.
+  validations:
+    required: true
+- type: checkboxes
+  attributes:
+    label: Checklist
+    options:
+    - label: If possible, I've reproduced the issue using the `main` branch of this package.
+      required: false
+    - label: This issue hasn't been addressed in an [existing GitHub issue](https://github.com/apple/swift-docc/issues).
+      required: true
+- type: textarea
+  attributes:
+    label: Expected Behavior
+    description: Describe what you expected to happen.
+  validations:
+    required: false
+- type: textarea
+  attributes:
+    label: Actual behavior
+    description: Describe or copy/paste the behavior you observe.
+  validations:
+    required: false
+- type: textarea
+  attributes:
+    label: Steps To Reproduce
+    description: |
+      Explanation of how to reproduce the incorrect behavior. 
+      
+      This could include a reduced version of your documentation bundle, or a link to the documentation (or code) that is exhibiting the issue.
+    placeholder: |
+      1. In this environment...
+
+      2. With this config...
+      
+      3. Run '...'
+      
+      4. See error...
```

Comment:
It looks like there's a Safari-specific bug where newlines in placeholders aren't respected: https://github.com/github-community/community/discussions/4290.

<img width="872" alt="Screen Shot 2022-07-02 at 9 17 54 AM" src="https://user-images.githubusercontent.com/6750147/177008340-1a1d5907-b49a-4e67-9f36-4413c83d3950.png">

Maybe we should replace this with a single line until that issue is resolved?

```suggestion
      1. In this environment... 2. With this config... 3. Run '...' 4. See error...
```

That will at least allow us to put spaces between each item.

Reply:
<img width="876" alt="image" src="https://user-images.githubusercontent.com/43724855/177021794-1d59634d-9dc7-4eb6-a375-195ba6483157.png">

Looks like this is resolved on Safari 16.0 (18614.1.16.11.3)

I can't reproduce this on my Mac with macOS Beta2(22A5286j) and Safari 16.0 (18614.1.16.11.3)

And it indeed shows the wrong placeholder on Safari 15.0 on my iPad with iPadOS 15.5

Reply:
Oh interesting- nice find. Glad to hear it’s been fixed! Okay I’m happy with this either way then- whichever formatting you prefer.